README.md

```diff
@@ -221,16 +221,86 @@ root
 </pre>
 
 
-
-## Usage
-Todo
-
-### Compiling
-
-- Install go version > 1.8
-
-### Compiling
-
+## Getting Started
+The following section describes building the reference implementation of the Obscuro protocol, running the unit and 
+integration tests, and deploying a local testnet for end-to-end testing. The reference implementation of Obscuro is 
+written in [go](https://go.dev) with the IDE of choice being [GoLand](https://www.jetbrains.com/go/) from Jetbrains. 
+Unless otherwise stated, all paths stated herein are relative to the root of the `go-obscuro` checkout.
+
+
+### Dependencies
+The following dependencies are required to be installed locally;
+
+- [go](https://go.dev) (version > 1.8)
```

Comment:
no. I have 1.18 locally and it works fine.
I tried 1.17 and lost a whole day.

Reply:
I remeber, soo frustrating, I think it was an installation/setup issue, 1.18 is easier to setup on mac because it's the [current stable version](https://formulae.brew.sh/formula/go) and backward compatible with 1.17.

The `go.mod` is @ 1.17, ego is also @ 1.17. And my only concern with the README is stating 1.18, is it will induce people to think that they can use 1.18 features (such as generics) only to realize that it won't work.

I guess if that's clear on the README, then 1.18 is also fine :)

Reply:
@tudor-malene @otherview Shall I leave this as > 1.8 then, or should I add a section that states than when running eGo you should use 1.17 as an addition?

Reply:
thanks, Moray. Yes, let's do that. Let's mention that the language features we use are 1.17, but we recommend having >=1.18 installed

Reply:
thanks ... done